tests/sles4sap/sys_param_check.pm

```diff
@@ -36,7 +36,7 @@ sub add_softfail {
     my ($module, $os_version, $bsc_number, @parameters) = @_;
     foreach my $parameter (@parameters) {
         if (check_var("VERSION", $os_version) && check_failure($module, $parameter)) {
-            record_soft_failure "$bsc_number - Wrong value for $parameter";
+            record_info('Softfail', "$bsc_number - Wrong value for $parameter", result => 'softfail');
```

Comment:
We need to use `record_soft_failure` here to mark the whole job as soft failed and `record_info` will not do that. As can be seen, the message to `record_soft_failure` does include a bsc# (via a variable). Is there a way to include this again?

Reply:
hmm, I can add exception for this dedicated string "bsc_number"